frappe_slack_connector/api/test_slack_channel.py

```diff
@@ -7,7 +7,7 @@
 
 
 @frappe.whitelist()
-def test_channel(channel_id: str = None):
+def test_channel(channel_id: str = ""):
```

Comment:
similarly, there is a check `if channel_id is None:`, better to just update type annotation